deputy_project_leads.md

```diff
@@ -38,6 +38,20 @@ This include checking that:
   the existing functions
 
 
+## Principle Engineer
+
+Matplotlib relies on wide and deep code base to implement its public API; the
+low-level details need to be correct to faithfully implement that API.
+In contrast to the API Consistency Leader, who is responsible for what
+the library does, the Principle Engineer is responsible for how. They
+are the point of contact for:
```

Comment:
I have learned many principles from @anntzer, but regardless suggest we nominate him for "Principal Engineer"  

```suggestion
## Principal Engineer

Matplotlib relies on wide and deep code base to implement its public API; the
low-level details need to be correct to faithfully implement that API.
In contrast to the API Consistency Leader, who is responsible for what
the library does, the Principal Engineer is responsible for how. They
are the point of contact for:
```